src/configfinder.coffee

```diff
@@ -86,11 +92,16 @@ expandModuleNames = (dir, config) ->
 
     config
 
-extendConfig = (config) ->
+extendConfig = (dir, config) ->
     unless config.extends
         return config
+    try
+        parentConfig = require resolve config.extends,
```

Comment:
if we cant resolve the path relative to our present directory, *then* we can look in node_modules